pkgs/top-level/all-packages.nix

```
@@ -23491,9 +23499,15 @@ with pkgs;
   libzra = callPackage ../development/libraries/libzra { };
 
   # requires a newer Apple SDK
-  zig = darwin.apple_sdk_11_0.callPackage ../development/compilers/zig {
+  zig_0_9_1 = darwin.apple_sdk_11_0.callPackage ../development/compilers/zig/0.9.1.nix {
```

```suggestion
   zig_0_9 = darwin.apple_sdk_11_0.callPackage ../development/compilers/zig/0.9.nix { 
```

We are not keeping previous 0.9.x versions, so we don't need to include this in the expression name.